internal/runnerinstall/runnerinstall.go

```diff
@@ -52,6 +52,9 @@ type InstallOpts struct {
 
 	// Unique ID for the runner.
 	Id string
+
+	// Flags which will be supplied to the `waypoint runner agent` command
+	RunnerRunFlags []string
```

Comment:
nit: could call this `RunnerAgentFlags` (since `ServerRun` is the command the flags are for in the case of `ServerRunFlags`)